doc/example/example_MPIPool.py

```diff
@@ -0,0 +1,77 @@
+import pypesto
```

Comment:
Please add file-level docstring what this script is supposed to demonstrate.

Reply:
docstrings always with double quotes